packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -33,8 +33,8 @@ const Map<String, String> kManuallyPinnedDependencies = <String, String>{
   'flutter_template_images': '4.2.0', // Must always exactly match flutter_tools template.
   'video_player': '2.2.11',
   // Keep pinned to latest until 1.0.0.
-  'material_color_utilities': '0.5.0',
-  // https://github.com/flutter/flutter/issues/115660
+  'material_color_utilities': '0.8.0',
+  // https://github.com/flutter/flutter/issues/111304
```

Comment:
oh wait, this looks wrong?

Reply:
I'm guessing a the diff resolution algorithm picked the wrong one

Reply:
yup! will fix!